doc/man3/SSL_CTX_set1_curves.pod

```diff
@@ -68,6 +70,12 @@ order. It can return zero if the client did not send a supported groups
 extension. If a supported group NID is unknown then the value is set to the
 bitwise OR of TLSEXT_nid_unknown (0x1000000) and the id of the group.
 
+SSL_get0_iana_groups() returns the set of IANA groups sent by a client
+in the supported groups extension. It returns the total number of
+supported groups. The contents of B<*out> is an array of integers
+holding the numerical value of the supported groups in the order they appear in
+the ClientHello. the function will return the supported groups length.
```

Comment:
```suggestion
SSL_get0_iana_groups() retrieves the list of groups sent by the
client in the supported_groups extension.  The B<*out> array of bytes
is populated with the host-byte-order representation of the uint16_t group
identifiers, as assigned by IANA.  The group list is returned in the same order
that was received in the ClientHello.  The return value is the number of groups,
not the number of bytes written.
```
"set of IANA groups" is a suboptimal term in at least two ways -- it's an ordered list, not an unordered set, and "IANA group" is not a common phrase; we instead refer to the group identifier as assigned or allocated by IANA.

It's also critical to indicate that the output array is to be treated as an array of 16-bit integers, and what byte order is used to encode the 16-bit values into the array of bytes.

Reply:
done with thanks.